### ISSUE TITLE
Options vanish at higher zoom levels

### ISSUE THREAD

**Issue by karlgroves** [WordPress/gutenberg#15286]
Options vanish at higher zoom levels

-   **Severity:** Medium
-   **Affected Populations:**
    -   Low-Vision
    -   Cognitively Impaired
-   **Platform(s):**
    -   All / Universal
-   **Components affected:**
    -   Edit Media

#### Issue description

On the Edit Media page there are two buttons ("Screen Options" and
"Help") which offer additional settings and information. These buttons
are missing when the page is enlarged with browser zoom above 150%.

Users who use browser zoom to enlarge text and clickable areas may not
realise there are options available to add media metadata such as
captions and descriptions, nor that there's an explanatory overview.

##### Issue Code

```
    @media screen and (max-width: 782px) {


        #collapse-menu,


        #screen-meta,


        #screen-meta-links,


        .post-format-select {


            display: none!important;


        }


    }
```

#### Remediation Guidance

Using the same media query, find another suitable design to retain these
options and keep them visible, or offer an alternative method of
reaching these options.

#### Relevant standards

-   [1.4.10 Reflow (Level
    AA)](https://www.w3.org/TR/WCAG20/#visual-audio-contrast-without-color)

**Note:** This issue may be a duplicate with other existing accessibility-related bugs in this project. This issue comes from the Gutenberg accessibility audit, performed by [Tenon](https://www.tenon.io) and funded by [WP Campus](https://wpcampus.org/). This issue is GUT-79 in Tenon's report

**Comment by afercia** [comment]
"Screen Options" and "Help" are part of the WordPress core admin pages. Regardless, and as far as I can tell, this was fixed in WordPress 5.1 see https://core.trac.wordpress.org/changeset/42644.

Please do feel free to reopen if the issue still applies on latest WordPress (5.2 will land in a few days).
